nutri-track-bot/src/main/java/com/nutritrack/bot/service/telegram/factory/TelegramFactory.java

```diff
@@ -47,11 +51,17 @@ public String getBotPath() {
                 return path;
             }
 
-            private void registerCommands() {
+            private void registerCommands(String languageCode) {
                 try {
+                    String commandsFile = "commands_en.json";
+
+                    if ("ru".equals(languageCode)) {
```

Comment:
The same [comment ](https://github.com/balmasov/nutri-track-bot/pull/6/files#r1673227397)for this variable "ru".